go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/descope/go-sdk
 
-go 1.18
```

Comment:
I think for SDKs we should support as far back as we can

Reply:
You might be right here. I kind of jumped the gun because I saw renovate trying and failing.